charts/drupal/files/settings.silta.php

```diff
@@ -5,6 +5,8 @@
  * Automatically injected settings for the Silta environment.
  */
 
+use Drupal\Core\Installer\InstallerKernel;
```

Comment:
1. Does this work with D8?
2. Will there be any performance penalties because of this include?

Reply:
@Jancis This is specifically for D8. As far as I know, no performance penalties.

Reply:
Whoops, meant D7. We'll need another file for D7.